deploy/olm-catalog/kubernetes-imagepuller-operator/kubernetes-imagepuller-operator.package.yaml

```diff
@@ -2,4 +2,4 @@ packageName: kubernetes-imagepuller-operator
 defaultChannel: stable
 channels:
   - name: stable
-    currentCSV: kubernetes-imagepuller-operator.v0.0.9
+    currentCSV: kubernetes-imagepuller-operator.v0.0.10
```

Comment:
I think this should stay at v0.0.9 since it is the latest version

Reply:
Yes, good catch.